eth/catalyst/api.go

```diff
@@ -298,6 +298,10 @@ func (api *ConsensusAPI) NewPayloadV1(params beacon.ExecutableDataV1) (beacon.Pa
 		log.Warn("Invalid timestamp", "parent", block.Time(), "block", block.Time())
 		return api.invalid(errors.New("invalid timestamp")), nil
 	}
+	if !api.eth.BlockChain().HasBlockAndState(block.ParentHash(), block.NumberU64()-1) {
+		log.Warn("State not available, ignoring new payload")
```

Comment:
Please add `api.remoteBlocks.put(block.Hash(), block.Header())` so that a fork choice update later might still work correctly.